lib/model/migrations/20171010-01-auth.js

```diff
@@ -0,0 +1,79 @@
+
+const up = (knex, Promise) => {
+  const createActees = knex.schema.createTable('actees', (actees) => {
+    actees.string('id', 36).primary();
+    actees.string('species', 36);
+  });
+  const createActors = knex.schema.createTable('actors', (actors) => {
+    actors.increments('id');
+    actors.enu('type', [ 'system', 'user', 'group', 'proxy' ]).notNull();
+    actors.string('acteeId', 36).notNull();
```

Comment:
This means that actors can also be actees, right? What's the use case for that?

Reply:
Users. I can potentially change a user's permissions.